.pipelines/pullrequest.yml

```diff
@@ -176,3 +189,33 @@ extends:
             displayName: Build snp with Docker
             env:
               PLATFORM: snp
+
+    - stage: docker
+      dependsOn:
+      - setup
+      - virtual_build
+      - snp_build
+      jobs:
+      # Build one image per each platform and push it to ACR
+      
+      # Virtual
+      - template: .pipelines/obpdocker.yml@self
+        parameters:
+          Name: scitt_virtual
+          DisplayName: Build Virtual Docker image
+          AcrName: 'confidentialledgeracrstaging'
+          DockerTag: $(gitTagOrCommit)_$(Build.BuildNumber)
```

Comment:
Please update the tags to make sure that the image is from a PR, e.g. `$(gitTagOrCommit)_$(Build.BuildNumber)_pr` if you still need to push the images.

Reply:
Addressed in https://github.com/microsoft/scitt-ccf-ledger/pull/298/commits/8d8590b86f438ebe81b05d19f7cbc880437c55a6